src/SignalRServiceExtension/SignalRMultiConnectionInfoAttribute.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.Azure.WebJobs.Description;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+    [AttributeUsage(AttributeTargets.ReturnValue | AttributeTargets.Parameter)]
+    [Binding]
+    public class SignalRMultiConnectionInfoAttribute : Attribute
```

Comment:
maybe discuss the naming later. already use `Info`... but we should not use abbr in naming for public usage (by @vicancy )

Reply:
The properties inside the class do not show anything related to "multi" though..